extra/docs_landing.html

```diff
@@ -1,238 +1,366 @@
-            <li class="listitem"><a href="https://cloud.elastic.co/pricing">Review pricing using our calculator</a></li>
-          </ul>
-        </div>
-      </td>
-    </tr>
-  </table>
+      </a>
+    </div>
+  </div>
 
-  <h3 style="margin-top: 0px !important">Need help?</h3>
-  <table width="100%;margin-bottom: 5px !important;">
-    <tr>
-      <td style="width: 50%; padding-top: 0px !important;">
-        <div class="itemizedlist">
-          <ul class="itemizedlist" type="disc">
-            <li class="listitem"><a href="en/welcome-to-elastic/current/get-support-help.html">How to get support</a></li>
-            <li class="listitem"><a href="https://discuss.elastic.co/">Forums</a></li>
-            <li class="listitem"><a href="https://www.elastic.co/training/">Training</a></li>
-            <li class="listitem"><a href="en/welcome-to-elastic/current/troubleshooting-and-faqs.html">Troubleshooting and FAQs</a></li>
-          </ul>
-        </div>
-      </td>
-      <td style="width: 50%; padding-top: 0px !important;">
-        <div class="itemizedlist">
-          <ul class="itemizedlist" type="disc">
-            <li class="listitem"><a href="https://www.elastic.co/events/?tab=1&solution=null&event=Meetup&region=null&language=English&industry=null">Meetups and virtual events</a></li>
-            <li class="listitem"><a href="https://www.elastic.co/videos/">Webinars</a></li>
-            <li class="listitem"><a href="https://ela.st/slack">Community Slack</a></li>
-          </ul>
+  <div class="row my-4">
+   <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-endpoint-security.html">
+      <div class="card h-100">
+        <h4 class="mt-3">
+          <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt5e0e0ad9a13e6b8c/634d9da18473831f96bbdf1e/security-logo-color-32px.png');"></span>
+          Protect my environment
+        </h4>
+        <p>Learn how to protect your hosts with endpoint threat intelligence from Elastic Security.</p>
+      </div>
+    </a>
+  </div>
+    <div class="col-md-4 col-12 mb-2">
+      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-general-purpose.html">
+        <div class="card h-100">
+          <h4 class="mt-3">
+            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltcb0e79820c355263/636c1905cbd6b84ecb4851b2/stack-logo-color-32px.png');"></span>
+            Build a custom solution
+          </h4>
+          <p>Learn how to deploy your own platform to store, search, and visualize any data.</p>
         </div>
-      </td>
-    </tr>
-  </table>
-</div>
+      </a>
+    </div>
+  </div>
+
+  <div class="my-5">
```

Comment:
I think reducing the margin around the my-5 class also makes sense.

<img width="856" alt="Screen Shot 2022-11-16 at 11 59 22 AM" src="https://user-images.githubusercontent.com/5618806/202281922-a38521e9-d738-49fa-9c1a-f1408a311f30.png">